app/controllers/worldwide_organisations_controller.rb

```diff
@@ -23,8 +23,36 @@ def show
     end
   end
 
+  def show_b_variant
+    @world_location = WorldLocation.with_translations(I18n.locale).find(params[:world_location_id])
+    @embassy_data = embassy_test_data(params[:id])
+    @primary_role = primary_role
+    @other_roles = ([secondary_role] + office_roles).compact
+
+    set_meta_description(@worldwide_organisation.summary)
+    set_slimmer_organisations_header([@worldwide_organisation] + @worldwide_organisation.sponsoring_organisations)
+
+    if show_b_variant? && @embassy_data
```

Comment:
Ah yes, I see! For some reason I though that `is_under_test?` also checks whether the organisation is included in YAML but such appears not to be the case!